x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/stack_alerts_page.ts

```diff
@@ -129,10 +140,11 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           firstSolutionFilter = quickFilters
             .filter((_: number, f: any) => f.attribs['data-test-subj'].endsWith('rule types'))
             .first();
-          expect(firstSolutionFilter).to.not.be(null);
+
+          expect(typeof firstSolutionFilter?.attr('data-test-subj')).to.be('string');
```

Comment:
Hey! What is the reasoning behind checking the `data-test-subj` if it is a string?

Reply:
If you look at the error here: https://github.com/elastic/kibana/issues/184882

```TypeError: Cannot read properties of undefined (reading 'replace')
    at selectorToTerms (test_subj_selector.ts:11:6)
    at subj (test_subj_selector.ts:63:10)
    at TestSubjects.click (test_subjects.ts:168:63)
    at Context.<anonymous> (stack_alerts_page.ts:140:28)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.apply (wrap_function.js:73:16)
```
The test is failing because this `attr('data-test-subj')` doesn't return a string (returns `undefined`) instead. So inside the `retry` im ensuring that the attribute we are asserting actually exists.

Reply:
I see thanks!